### ISSUE TITLE
Progress information must not be provided only by color in Progress tab

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1941]
URL: http://clarity-ks.surge.sh/progress-bars

Tools: Web Accessibility Toolbar (WAT2.0) 

Reproduced Steps:
1. Login Clarity Kitchen Sink with web browser.
2. Go to Progress tab.
3. Use WAT (Colour - Greyscale) where content may be color dependent to determine if the meaning is clear when the color is removed in Progress tab.

Actual Results:
Color is the only means to convey information for Stacked Layout and Stacked and resized layout in Progress tab.
For more details, please refer to "Progress-InfomationIsProvidedByColor.png".

Expected Results:
Color cannot be the only means to convey information.
Information conveyed through color must also be provided in text on the screen.

Fix Suggestion:
The same information is represented textually on the screen or through other visual (non-color) differentiation, such as Shape, Position or Size.

Violation of Section 508 Standards: 
Section 1194.22(c) Web pages shall be designed so that all information conveyed with color is also available without color, for example from context or markup.

User Impact: Blind and color blind.

Screen shot:
1.	Progress-InfomationIsProvidedByColor.png

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/85d9774b-5d7d-47f1-bac0-d36a57e2e93e)

NOTE: Given that we have progress bars (even ones called out in the screenshot) with the percentages right next to them, I'm inclined to file this one as an implementation detail, not a bug.

**Comment by mathisscott** [comment]
Percents are listed right next to three of the progress bars in the screenshot.

We will need to make an a11y recommendation re: accessible progress bars in our dox.
